build.sbt

```diff
@@ -621,6 +621,18 @@ lazy val zioJson: ProjectMatrix = (projectMatrix in file("json/zio"))
   )
   .dependsOn(core)
 
+lazy val weePickleJson: ProjectMatrix = (projectMatrix in file("json/weepickle"))
```

Comment:
this should also be added to aggregates so that the subproject is included in the root build :)